### DISCUSSION TITLE
Loading local fonts (.ttf & .otf)

### DISCUSSION THREAD

**Discussion by MalacTheLittle** [airsdk/Adobe-Runtime-Support#1448] [Ideas]
Is there any chance to make this possible? 
In my case I need it for ePub reader project.

**Comment by itlancer** [comment]
May be related feature request: https://github.com/airsdk/Adobe-Runtime-Support/issues/149

**Reply by MalacTheLittle** [reply]
You're right, I missed that one. Since it was posted 2 years ago, here's a little reminder :)
Also, I need it for use in `flash.text.engine`.

**Reply by ajwfrost** [reply]
@MalacTheLittle just to confirm what you mean here? If the fonts are locally installed, you can just use a device font?

**Reply by MalacTheLittle** [reply]
@ajwfrost I was thinking of loading not installed local fonts. In my case, [ePub](http://idpf.org/epub/30/) can contain fonts used in publication and I should be able to load em and render publication text with them.

Another issue is that fonts included in ePub can be (usually are) obfuscated and/or subsetted, wich means I should be able to load fonts bytearray. Here's some [info](http://idpf.org/epub/30/spec/epub30-ocf.html#font-obfuscation) about obfuscation.

**Comment by MalacTheLittle** [comment]
@ajwfrost Any updates on this?
